### DISCUSSION TITLE
Consuming too much power in idle

### DISCUSSION THREAD

**Discussion by astro-ray** [fast4x/RiMusic#3739] [General]
Since last couple of months, I noticed my phone is draining too much battery power when left idle. Testing a few things, it turns out RiMusic was consuming a lot of power. After uninstalling it, the power drain is not as bad.
This app doesn't do anything for it to draining power like this, it could be something else that I might have screwed up.

**Comment by fast4x** [comment]
Use app in minimal mode, disable all effects and use essential player, timeline etc...
